app/src/main/java/ch/epfl/skysync/navigation/Route.kt

```diff
@@ -0,0 +1,11 @@
+package ch.epfl.skysync.navigation
+
+object Route {
+  const val MAIN = "Main screen"
```

Comment:
I'm not sure I understand what the `Main screen` represent

Reply:
"Main screen" is maybe not quit a good name... It is in fact the graph route that contains the four screens : Home, Flight, Chat and Calendar.  As it is a nested graph to access to these four screens from outside (in our case from the login) we need to navigate to this MAIN node which will automatically redirect to the Home screen

Reply:
Okay, that makes sense, then "Main graph" would be clearer to me